src/server.js

```diff
@@ -76,8 +76,8 @@ export default async function start({
       json: halson,
     })));
 
-    await app.start();
     console.info('Server running at:', app.info.uri); // eslint-disable-line
```

Comment:
I think the console message is redundant now. Let's change it to something like this...
```
console.info('Server setup completed...', 'Start the server.');
```

Reply:
Alright